site/content/recipes/explain_code_structure.md

```diff
@@ -0,0 +1,52 @@
+# Objective
+
+Reviewer has a good understanding of the overall structure of the
+code. When reviewing a section of the code, they have an idea of its
+purpose and how it relates to other parts.
+If the section to review is not yet defined, they are in a position to
+help the author choose it.
+
+# Steps
+
+1. The author describes the structure of the code to the reviewer(s).
```

Comment:
I think the addition of the word "overall" will help make it clear we are talking about the entirety of the code/program, rather than the section that will be reviewed. You use this word below anyway.

```suggestion
1. The author describes the overall structure of the code to the reviewer(s).
```

Reply:
How about emphasizing the objective and identifying the "steps" as suggestions to get there?